README.rst

```diff
@@ -62,6 +62,11 @@ You can pass many options to the configure script; run ``./configure --help``
 to find out more.  On macOS and Cygwin, the executable is called ``python.exe``;
 elsewhere it's just ``python``.
 
+If you are running on MacOS with the latest updates installed, make sure to install
```

Comment:
Sorry to be pedantic but the correct current styling according to Apple is  "macOS", not "MacOS", and that's the way it appears elsewhere in README.rst.